### PR TITLE
adds updated text to homepage. adds yellow headers and links to factoids

### LINKED ISSUE #71: Edit Home Page Text
The United States 2020 Census 
When you respond, we all benefit. 
 
Answer Confidentially 
Online, by Phone, or by Mail 
 

The 2020 Census is here. Your participation helps ensure our community will have access to housing, healthcare, schools, community programs, better transportation, and government representation. 

Every 10 years, the U.S. Census Bureau sets out to count every person living in the United States — regardless of age, citizenship status, and gender. Getting the 2020 Census right is important for all our communities — particularly those most likely to be undercounted. A 2020 Census undercount could put billions of federal dollars and congressional representation for California at risk!

Complete the census survey **online, by phone or mail**. The process is easy, quick and confidential! All you need is the ID code mailed to you by the U.S. Census Bureau, or your mailing address. Remember, to _include everyone living and sleeping in your house_. That means babies, children, teens, roommates, etc. Click the "Take the Census Now" button to get started!

### PR DESCRIPTION
**Pull request by ghost**
resolves #71 . adds updated text to homepage. adds yellow headers and links to factoids. still needs icon images. also updates translations for new text.

### CODE REVIEW

**Review by JMStudiosJoe — APPROVED** [review]

**Comment by JMStudiosJoe** [comment]
@SeanKier plz fix conflicts then will merge

**Comment by ghost** [comment]
@JMStudiosJoe thanks. updated to fix conflicts

File: i18n/translations/translations.en.json [hunk 1]
```diff
@@ -452,6 +452,12 @@
...
+    "translation": "Yes. The online survey is available in 12 languages (Spanish, Chinese, Vietnamese, Korean, Russian, Arabic, Tagalog, Polish, French, Haitian Creole, Portuguese, Japanese); phone assistance will also be available in these languages by calling (####). The paper survey is available in English and Spanish. Call to request a paper survey./n The U.S. Census Bureau will provide other materials such as languages guides, glossaries, etc. in 59 languages. Residents who need assistance or who do not have computer or internet access can visit a City library or community center for assistance."
     "english": "I don’t speak English. Will the Census survey be available in other languages?",
```
Comment (turnerhayes) [inline i18n/translations/translations.en.json]: Results in invalid JSON--possibly a merge conflict resolution error?

**Review by turnerhayes — CHANGES_REQUESTED** [review]
Fix broken JSON and then it can be merged.

File: i18n/translations/translations.en.json (hunk 1 above)
Comment (ghost) [reply i18n/translations/translations.en.json]: thanks. fixed json errors

**Comment by ghost** [comment]
Think it should be good to go now

**Review by JMStudiosJoe — APPROVED** [review]

**Comment by JMStudiosJoe** [comment]
@turnerhayes  @SeanKier  is this ready to go? I am not able to merge.

**Review by turnerhayes — APPROVED** [review]

**Comment by turnerhayes** [comment]
Thanks for reminding me--it's good to go from my end, but the merge conflict still needs to be resolved.

**Comment by ghost** [comment]
Just resolved the conflict. So should be good to merge now.

**Comment by turnerhayes** [comment]
Thanks Sean! Merged.

**Pull request merged**